### DISCUSSION TITLE
should "no recent transactions" state show options to add SOL?

### DISCUSSION THREAD

**Discussion by wentokay** [coral-xyz/backpack#3251] [General]
I'm not sure if this is an intentional design choice, if it is then please ignore! I was wondering if it should it show some options for adding SOL to the wallet instead as you wouldn't be able to install any xNFTs without it?

<img width="367" alt="Screenshot 2023-03-09 at 09 08 37" src="https://user-images.githubusercontent.com/101902546/224102773-d1657d5e-9067-41e4-b76b-47df44f6ed3a.png">

**Comment by armaniferrante** [comment]
Good idea!
